doc/src/site/sphinx/design/spark_h2o_mapping.rst

```diff
@@ -67,3 +67,7 @@ As type ``T``, we support the following types:
 +--------------------------------------------------+
 
 As is specified in the table, Sparkling Water provides support for transforming arbitrary scala class extending ``Product``, which are, for example, all case classes.
+
+.. rubric:: Footnotes
+.. [1] The H20 type is String if cardinality is greater than 10 000 0000 or ratio of unique values to all values is 95% at least.
+.. [2] The possible values are "1" and "0" for true and false respectively.
```

Comment:
nit: Could we mention in the sentence that "1" and "0" belong to the categorical column of H2OFrame?